src/api/modules/auth/index.ts

```diff
@@ -2,9 +2,9 @@ import axiosInstance from 'api/core';
 
 import { type LoginRequest, LoginResponse, RefreshRequest, RefreshResponse } from 'api/modules/auth/types';
 import { type CommonResponseReturnType } from 'api/modules/commonType';
+import { LogoutRequest } from './types';
```

Comment:
요기도 type import 되면 좋겠네요! 
```suggestion
import type { LogoutRequest } from './types';
```